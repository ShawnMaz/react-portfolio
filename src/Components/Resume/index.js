import resume from './resume.pdf';

function Resume(){

    return(
        <section className='container'>
            <div className='row'>
                <div className='col-sm-12 mx-auto my-3'>
                    <p><a href={resume} className='text-primary' download>Click here</a> to download the resume.</p>
                </div>
                <div className='col-sm-12'>
                    <h2>Front-end Profeciencies</h2>
                    <ul>
                        <li>HTML</li>
                        <li>CSS</li>
                        <li>Responsive design</li>
                        <li>React</li>                        
                        <li>Bootstrap</li>                        
                    </ul>
                </div>
                <div className='col-sm-12'>
                    <h2>Back-end Profeciencies</h2>
                    <ul>
                        <li>Node</li>
                        <li>Express</li>
                        <li>MySQL, Sequelize</li>
                        <li>MongoDB, Mongoose</li>
                        <li>REST</li>
                    </ul>
                </div>
            </div>
        </section>
    );
}

export default Resume;