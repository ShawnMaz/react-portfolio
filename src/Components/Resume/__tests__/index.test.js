import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom';
import Resume from '..';

afterEach(cleanup);

describe('Resume component', () => {
    it('renders', () => {
        render(<Resume/>);
    });

    it('matches snapshot', () => {
        const {asFragment} = render(<Resume/>);
        expect(asFragment()).toMatchSnapshot();
    });
});