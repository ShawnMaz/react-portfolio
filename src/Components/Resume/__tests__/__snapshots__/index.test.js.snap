// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component matches snapshot 1`] = `
<DocumentFragment>
  <section
    class="container mt-3"
  >
    <h2>
      Resume
    </h2>
    <div
      class="row"
    >
      <div
        class="col-sm-12 mx-auto my-3"
      >
        <p>
          <a
            class="text-primary"
            download=""
            href="resume.pdf"
          >
            Click here
          </a>
           to download the resume.
        </p>
      </div>
      <div
        class="col-sm-12"
      >
        <h2>
          Front-end Profeciencies
        </h2>
        <ul>
          <li>
            HTML
          </li>
          <li>
            CSS
          </li>
          <li>
            Responsive design
          </li>
          <li>
            React
          </li>
          <li>
            Bootstrap
          </li>
        </ul>
      </div>
      <div
        class="col-sm-12"
      >
        <h2>
          Back-end Profeciencies
        </h2>
        <ul>
          <li>
            Node
          </li>
          <li>
            Express
          </li>
          <li>
            MySQL, Sequelize
          </li>
          <li>
            MongoDB, Mongoose
          </li>
          <li>
            REST
          </li>
        </ul>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
